test/abort-task.ts

```diff
@@ -1,6 +1,6 @@
 import { AbortController } from 'abort-controller';
 import { EventEmitter } from 'events';
-import Piscina from '..';
+import Piscina = require('..');
```

Comment:
Let's tackle this differently, by exporting `Piscina` in different manners by exporting it as `default`, and `named export`.

Otherwise, this can potentially break the current user's expectations if `esModuleInterop` is set to `false`. Wdyt?

Reply:
does the default export created by TS work as an actual default export in Node? The current one does as it's just `module.exports`

Reply:
If exported by default, it will export it as `module.exports.default = Piscina`, if by named export, it translates to `module.Piscina = Piscina`. If just used as `export Piscina`, it just does `module.exports = Piscina`.

Exporting it as default, and named will enable imports like:

- `import { Piscina } from 'piscina'`
- `import Piscina, { SomeType } from 'piscina'`

Reply:
seems #517 did this 👍